.changeset/little-flies-unite.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-extension-dsl-data-space': patch
```

Comment:
Not sure if this should be separated into a different changeset or not 😓

Reply:
I think, the change in this PR should only happen in `dsl-data-space`